db/migrate/20190712135032_set_provisioned_state_to_services.rb

```diff
@@ -0,0 +1,14 @@
+class SetProvisionedStateToServices < ActiveRecord::Migration[5.1]
+  class Service < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Update service lifecycle state values") do
+      Service.in_my_region.where(:lifecycle_state => nil, :retired => nil).each do |service|
```

Comment:
I think it can be  1 query 

```
Service.in_my_region.where(:lifecycle_state => nil, :retired => nil).update_all(:lifecycle_state => 'provisioned')
```